hs_core/tests/api/native/test_core_metadata.py

```diff
@@ -521,12 +521,12 @@ def test_coverage(self):
 
         self.res.metadata.coverages.get(type='box').delete()
         # now try to create point type coverage with invalid data
-        # value for 'east' should be >= -180 and <= 180 and 'north' >= -90 and <= 90
-        value_dict = {'east': '181.45678', 'north': '50', 'units': 'decimal deg'}
+        # value for 'east' should be >= -360 and <= 360 and 'north' >= -90 and <= 90
+        value_dict = {'east': '360.45678', 'north': '50', 'units': 'decimal deg'}
```

Comment:
It's worth checking the computed values after the metadata element is created.

Reply:
I have more tests related to spatial coverage here:
https://github.com/hydroshare/hydroshare/pull/4407/files#diff-71b4e76afaed1db95b36346c18255e9df9a26d3123b749698269454fb6e6acdcR526